cookiecutter/main.py

```diff
@@ -4,8 +4,11 @@
 The code in this module is also a good example of how to use Cookiecutter as a
 library rather than a script.
 """
+from __future__ import unicode_literals
```

Comment:
Any reason for adding this?

Reply:
good catch, it seems I added this because I needed it for an initial change. It's been over 1.5 years, and I guess I forgot about that. I'll remove it